micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/ProcessEngineFactory.java

```diff
@@ -50,7 +50,9 @@ public ProcessEngine processEngine(ProcessEngineConfiguration processEngineConfi
      * @throws IOException if a resource, i.e. a model, cannot be loaded.
      */
     private void deployProcessModels(ProcessEngine processEngine) throws IOException {
+        CamundaBpmVersion camundaBpmVersion = new CamundaBpmVersion();
         log.info("Searching non-recursively for models in the resources");
+        log.info("Camunda version: {}", camundaBpmVersion.getVersion());
```

Comment:
* please move this log statement to the beginning of the method `processEngine(ProcessEngineConfiguration processEngineConfiguration)`, i.e. before building the process engine.
* please change log message to "Camunda BPM version: {}"